datanode/gateway/graphql/schema.graphql

```diff
@@ -69,7 +69,7 @@ type Subscription {
     assetId: ID
     "Type of the account"
     type: AccountType
-  ): AccountUpdate!
+  ): [AccountUpdate!]!
```

Comment:
Interesting here it's a non nullable array of non nullable AccountUpdate, if you look at orders or trades, we get a nullable list of non nullable objects

Reply:
The difference, I believe, is that with

```[AccountUpdate!]!```

you get back an empty list if there is, well an empty list.

With 

[AccountUpdate!]

With an empty list, you seem to get back 'null', which seems like more work to handle on the other end to me but I don't really mind